brianlong/www.validators.app
# 182815760-improvements-in-daemons

#### What's this PR do?
- add missing indexes
- make minor improvements on daemons (see tasks in story)

#### How should this be manually tested?
- run migrations
- run tests
- run daemons (e. g. gather_rpc_mainnet_daemon.rb, validator_score_mainnet_v1_daemon.rb) and see if there are no errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182815760)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
